src/fundus/publishers/de/__init__.py

```diff
@@ -30,6 +31,17 @@
 
 # noinspection PyPep8Naming
 class DE(PublisherEnum):
+    SportSchau = PublisherSpec(
+        name="Sportschau",
+        domain="https://www.sportschau.de/",
+        parser=SportSchauParser,
+        sources=[
+            RSSFeed("https://www.sportschau.de/index~rss2.xml"),
+            Sitemap("https://www.sportschau.de/index~sitemap_p-0.xml"),
+            NewsMap("https://www.sportschau.de/kompakt-sp-100~news.xml"),
+        ],
```

Comment:
For some reason sportschau seems to also link articles from other websites such as mdr, which then naturally can't be parsed by your Parser. I would suggest adding `url_filter=inverse(regex_filter("sportschau.de"))` to your PublisherSpec. Don't worry if it doesn't seem to make a difference yet, this is due to a bug that is fixed with #459

Reply:
sorry, what does "inverse" do, because it seems like it does not work

Reply:
Inverse works like a logical not for filters. I did forget to mention you still have to import it though. Sorry about that:

`from fundus.scraping.filter import inverse, regex_filter`

Reply:
The PR looks good, I just forgot to mention that you have to regenerate the test files if you update the extraction logic: `scripts.generate_parser_test_files -p SportSchau -oj`